docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -6,7 +6,13 @@ POSDYON Population Synthesis Configuration Guide
 
 This documentation provides a detailed overview of the configuration options available in the Posydon software package.
 
-TODO: fill each table cell with a description of the parameter and the options
+
+.. warning::
+    The default values in the population_params.ini file included in POSYDON 
```

Comment:
Maybe this warning was always here, but I never realized that we are making a statement this strongly about the default values? If so then maybe I need to change the text about the warning_verbose, error_checking_verbose, history_verbose parameters because I explicitly wrote default=False, or wanted to imply that we recommend the user set those to False.

Reply:
For those parameters it's fine. This is mostly for science things, such as different remnant mass prescriptions, kicks, etc.

We don't want to imply that the default is what we always use or that that is the "best" for others/the user to use  :)